ambry-api/src/main/java/com/github/ambry/quota/QuotaResource.java

```diff
@@ -13,10 +13,14 @@
  */
 package com.github.ambry.quota;
 
+import com.github.ambry.account.Container;
+
+
 /**
  * Resource for which quota is specified for enforced.
  */
 public class QuotaResource {
```

Comment:
Just wondering if equals() and hashCode() methods of Object class should be overriden for HashSet to filter duplicates correctly.

Reply:
done.